regression-tests/CMakeLists.txt

```diff
@@ -119,8 +146,48 @@ function(cppfront_build_tests)
         "-Dcppfront_DIR=${cppfront_DIR}"
         "-Dcppfront-exe_DIR=${cppfront-exe_DIR}"
         "-DCPPFRONT_FLAGS=${ARG_EXTRA_FLAGS}"
+        # There's `CMAKE_CXX_LINKER_LAUNCHER`, too. So far, it's not needed.
+        "-DCMAKE_CXX_COMPILER_LAUNCHER=${CMAKE_COMMAND};-D;OUTPUT_FILE=${gen_cpp_src}.output;-P;../../ExecuteWithRedirection.cmake;--"
```

Comment:
These only work with "The Makefile Generators and the Ninja generator".
The alternative is [`CMAKE_EXPORT_COMPILE_COMMANDS`][].
I already handle that at <https://github.com/JohelEGP/jegp.cmake_modules/blob/master/modules/.detail/JEGPReadExportedCompileCommand.cmake>.

[`CMAKE_EXPORT_COMPILE_COMMANDS`]: https://cmake.org/cmake/help/latest/variable/CMAKE_EXPORT_COMPILE_COMMANDS.html

Reply:
I would still need to build and install.
So I'd compile twice to get the output.
Maybe I can hijack `CMAKE_CXX_COMPILER`
and use it like `CMAKE_CXX_COMPILER_LAUNCHER`.